src/python/lib/ensembl/compara/orthtools/orthofinder.py

```diff
@@ -0,0 +1,82 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Methods for preparing input files and running OrthoFinder.
+
+This module provides::
+
+    :func:`prepare_input_orthofinder()` which copies input fasta files for OrthoFinder
+            into a specified working directory
+
+    :func:`run_orthofinder()` which runs OrthoFinder
+
+Typical usage example::
+
+    from ensembl.compara.orthtools import prepare_input_orthofinder
+    prepare_input_orthofinder('path/to/source_dir', 'path/to/target_dir')
+
+    from ensembl.compara.orthtools import run_orthofinder
+    run_orthofinder('path/to/input/fasta/files')
+
+    from ensembl.compara.orthtools import run_orthofinder
+    run_orthofinder('path/to/input/fasta/files', 16, 4)
+
+"""
+
+__all__ = ['prepare_input_orthofinder', 'run_orthofinder']
+
+import os
+import subprocess
+
+def prepare_input_orthofinder(source_dir: str, target_dir: str) -> None:
```

Comment:
https://github.com/Ensembl/ensembl-compara/blob/feature/homology_annotation/scripts/pipeline/symlink_fasta.py <- just spotted this, and wanted to direct you to this existing script. Not saying you have to use it - and perhaps this does not meet the specs, but a notable difference is that here you duplicate the data by `rsync` -ing the directory whilst for orthofinder, you could use symlinks and save the duplication, OrthoFinder will just write to the directory given with symlinked files/dir and use the real paths to utilise the data.

Reply:
Great, thank you! I will swap copying files for creating symlinks 👍